ui/app/AppLayouts/Chat/components/GroupInfoPopup.qml

```diff
@@ -276,7 +277,15 @@ ModalPopup {
                         elide: Text.ElideRight
                         Layout.fillWidth: true
                         font.pixelSize: 13
+                        MouseArea {
+                          anchors.fill: parent
+                          cursorShape: Qt.PointingHandCursor
+                          onClicked: {
+                            popup.profileClick(model.userName, model.pubKey, model.identicon)
+                          }
+                        }
```

Comment:
Consider this a "quick-and-easy" win here. I think the entire group info popup needs rework and make use of `ContactList` and `Contact` components, so we don't reinvent the wheel.

However, to make that work, those two components need a bit of fine-tuning as well.